From 5c62fc59cf448096baea3f732143e6baa71a60f9 Mon Sep 17 00:00:00 2001
Message-ID: <5c62fc59cf448096baea3f732143e6baa71a60f9.1699294468.git.github@sicherha.de>
In-Reply-To: <9c6dfb6623d7a05ab7e51620aacf1e4be16cf157.1699294468.git.github@sicherha.de>
References: <9c6dfb6623d7a05ab7e51620aacf1e4be16cf157.1699294468.git.github@sicherha.de>
From: Christoph Erhardt <github@sicherha.de>
Date: Sun, 5 Nov 2023 23:09:22 +0100
Subject: [PATCH 2/3] Fix unit tests on i686 host

Make sure that the canonical name is used in all assignments to
`$MACHINE`.

Signed-off-by: Christoph Erhardt <github@sicherha.de>
---
 test/elf/common.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/elf/common.inc b/test/elf/common.inc
index 9ac3dcc5..3dc7126d 100644
--- a/test/elf/common.inc
+++ b/test/elf/common.inc
@@ -20,7 +20,7 @@ fi
 
 # Set tool names
 if [ -z "$TRIPLE" ]; then
-  MACHINE=$HOST
+  MACHINE=$(canonical_name "$HOST")
   TESTDIR=out/test/elf/$HOST
   CC="${TEST_CC:-cc}"
   CXX="${TEST_CXX:-c++}"
-- 
2.41.0

